bin/tests/bootstrap-integration.bats

```diff
@@ -0,0 +1,33 @@
+#!/usr/sbin/bats
+
+load 'test_helper/bats-support/load'
+load 'test_helper/bats-assert/load'
+load 'test_helper/bats-file/load'
+
+setup () {
+    TEST_TEMP_DIR="$(temp_make --prefix 'otomi-values-')"
```

Comment:
Before you joined we agreed on some coding standards, e.g.: all variables that are not coming from environment are lower case. 
Please read https://otomi.slack.com/archives/C011D78FP47/p1604919027008400

Most probably we need to add it somewhere in dev guide.

Reply:
This is the first time I see this document, I agree with this of course, I'll update this pull request and add these modifications. But yes, please consider adding this to the development documentation.

Reply:
Yes, I thought I had done so already. Excuse me